compliance/elasticsearch/pom.xml

```diff
@@ -101,8 +101,22 @@
 					<groupId>com.vividsolutions</groupId>
 					<artifactId>jts</artifactId>
 				</exclusion>
+				<exclusion>
+					<groupId>com.fasterxml.jackson.core</groupId>
+					<artifactId>jackson-databind</artifactId>
+				</exclusion>
+				<exclusion>
+					<groupId>com.fasterxml.jackson.core</groupId>
+					<artifactId>jackson-core</artifactId>
+				</exclusion>
 			</exclusions>
 		</dependency>
+		<dependency>
+			<groupId>org.yaml</groupId>
+			<artifactId>snakeyaml</artifactId>
+			<version>1.26</version>
+			<scope>test</scope>
+		</dependency>
```

Comment:
Maybe just exclude it from the dependency that has the wrong version?

Reply:
Good point, I excluded the jackson-databind-yaml dependency (which caused the snakeyaml issue) instead of explicitly including the specific (newer) version of snakeyaml

Reply:
:man_juggling: